packages/subgraph/templates/subgraph.template.yaml

```diff
@@ -20,11 +20,14 @@ dataSources:
       entities:
         - MACI
         - Poll
+        - Tally
       abis:
         - name: MACI
           file: ./node_modules/maci-platform-contracts/build/artifacts/contracts/maci/MACI.sol/MACI.json
         - name: Poll
           file: ./node_modules/maci-platform-contracts/build/artifacts/contracts/maci/Poll.sol/Poll.json
+        - name: Tally
+          file: ./abi/Tally.json
```

Comment:
why a local file and not from maci-platform-contracts?

Reply:
It's a workaround because subgraph doesn't support multidimensional arrays. So, it uses the abi from `maci-platform-contracts` and cleans these params because we don't use them. This is done in `precodegen` step (script `clean-abi`).